docs/modules/release-notes/pages/all-releases/4_16_1S.adoc

```diff
@@ -0,0 +1,188 @@
+= Kobiton 4.16.1S release notes
+:navtitle: Kobiton 4.16.1S release notes
+
+_November 22, 2024_
+
+== New OS support
+
+This release adds support for the following:
+
+* Android 15
+
+[NOTE]
+====
+During a Manual session, Android 15 devices may become unresponsive after one of the following actions:
+
+* Using Chrome or Gmail.
+* Typing on the virtual keyboard.
+
+To make the device responsive again, restart the device using the Restart button in the Device Details or the Device Management page (appropriate permission required).
+
+To reduce the occurrence of this issue, update *Play Store services*, *Chrome*, and *Gmail* on the device to the latest version.
+====
+
+* iOS/iPadOS 18.0.1
+
+[NOTE]
+For iOS/iPadOS 18.0.1, the cleanup policy does not clear signed-in accounts in the Mail app and Safari browser cache.
+
+* MacOS 15 Sequoia (*except* Standalone portal with self-signed SSL certificate)
+
+[WARNING]
+====
+
+Portal with a *self-signed SSL certificate* does not support Mac mini hosts with MacOS Sequoia. *Do not* update to Sequoia on the Mac mini host in this case until further notice.
```

Comment:
Portals that use a self-signed SSL certificate do not currently support Mac mini hosts with MacOS Sequoia.